Dockerfile

```diff
@@ -12,9 +19,10 @@ RUN pip install --upgrade pip && \
     python install-poetry.py --version $POETRY_VERSION
 ENV POETRY_VIRTUALENVS_CREATE="false"
 ENV POETRY_INSTALLER_PARALLEL="false"
+ENV VIRTUAL_ENV=/root/.venv
```

Comment:
for some reason poetry seems to be needing this in order to find the virtual env